strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -48,9 +44,8 @@
         mergeMode = TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS)
 @Execution(CONCURRENT)
 public class MavenProxyRepositoryProviderTestIT
-        extends TestCaseWithMavenArtifactGenerationAndIndexing
+        extends ArtifactResolutionServiceHelper
```

Comment:
@ptirador I mean that we should inject `ArtifactResolutionServiceHelper` here, no need to use inheritance for this. You can define `ArtifactResolutionServiceHelper` bean instance within `Maven2LayoutProviderCronTasksTestConfig`.